package.json

```diff
@@ -52,19 +52,13 @@
   },
   "homepage": "https://github.com/rollup/rollup",
   "dependencies": {
-    "@types/acorn": "^4.0.3",
-    "acorn": "^5.5.3",
-    "acorn-dynamic-import": "^3.0.0",
-    "locate-character": "^2.0.5",
-    "pretty-ms": "^3.1.0",
-    "signal-exit": "^3.0.2",
-    "date-time": "^2.1.0",
-    "is-reference": "^1.1.0",
-    "sourcemap-codec": "^1.4.1",
-    "require-relative": "^0.8.7",
-    "rollup-pluginutils": "^2.0.1"
+    "@types/estree": "0.0.38",
+    "@types/node": "*"
```

Comment:
Thanks, I missed the devDependency here actually (and for some reason my npm wasn't giving a warning on this).

Well EventEmitter has been there since the very start in Node so yes, any version. The star is so that it can work alongside any version a user may install.